<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>929</width>
    <height>757</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* 다크 테마 */
QMainWindow {
    background-color: #2b2b2b;
}

QListWidget {
    background-color: #1e1e1e;
    color: #ffffff;
    border: 1px solid #3c3c3c;
    border-radius: 5px;
    padding: 5px;
    font-size: 14px;
}

QListWidget::item:hover {
    background-color: #3c3c3c;
}

QListWidget::item:selected {
    background-color: #0d7377;
    color: white;
}

QTextEdit {
    background-color: #1e1e1e;
    color: #e0e0e0;
    border: 1px solid #3c3c3c;
    border-radius: 5px;
    padding: 10px;
    font-family: &quot;Yu Gothic&quot;, &quot;Meiryo&quot;;
    font-size: 13px;
}

QPushButton {
    background-color: #0d7377;
    color: white;
    border: none;
    border-radius: 5px;
    padding: 10px 20px;
    font-size: 14px;
    font-weight: bold;
}

QPushButton:hover {
    background-color: #14a085;
}

QPushButton:pressed {
    background-color: #0a5a5d;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">/* 다크 테마 */
QMainWindow {
    background-color: #2b2b2b;
}

QListWidget {
    background-color: #1e1e1e;
    color: #ffffff;
    border: 1px solid #3c3c3c;
    border-radius: 5px;
    padding: 5px;
    font-size: 14px;
}

QListWidget::item:hover {
    background-color: #3c3c3c;
}

QListWidget::item:selected {
    background-color: #0d7377;
    color: white;
}

QTextEdit {
    background-color: #1e1e1e;
    color: #e0e0e0;
    border: 1px solid #3c3c3c;
    border-radius: 5px;
    padding: 10px;
    font-family: &quot;Yu Gothic&quot;, &quot;Meiryo&quot;;
    font-size: 13px;
}

QPushButton {
    background-color: #0d7377;
    color: white;
    border: none;
    border-radius: 5px;
    padding: 10px 20px;
    font-size: 14px;
    font-weight: bold;
}

QPushButton:hover {
    background-color: #14a085;
}

QPushButton:pressed {
    background-color: #0a5a5d;
}</string>
   </property>
   <widget class="QListWidget" name="sentenceListWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>861</width>
      <height>291</height>
     </rect>
    </property>
    <property name="visible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QTextEdit" name="dictionaryTextEdit">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>310</y>
      <width>521</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="debugLogTextEdit">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>540</y>
      <width>521</width>
      <height>141</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string>Debug Log (개발용)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ankiButton">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>470</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>안키</string>
    </property>
   </widget>
   <widget class="QComboBox" name="processComboBox">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>320</y>
      <width>311</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="selectRoiButton">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>370</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>캡쳐 영역 선택</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startCaptureButton">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>430</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
      QPushButton {
        background-color: #14a085;
        color: white;
        font-weight: bold;
      }
      QPushButton:hover {
        background-color: #0d7377;
      }
      QPushButton:disabled {
        background-color: #555555;
      }
     </string>
    </property>
    <property name="text">
     <string>Start Capture</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stopCaptureButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>490</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
      QPushButton {
        background-color: #c0392b;
        color: white;
        font-weight: bold;
      }
      QPushButton:hover {
        background-color: #e74c3c;
      }
      QPushButton:disabled {
        background-color: #555555;
      }
     </string>
    </property>
    <property name="text">
     <string>Stop Capture</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>929</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
